### PR TITLE
Support Rails 4.x

### PR DESCRIPTION
**Pull request by abuiles**
I'd using eak with Rails 4.1 from my own fork, so far everything is working fine.

### CODE REVIEW

**Comment by bcardarella** [comment]
@rjackson is this OK? I know some of the generator stuff is Rails 4+, not sure about 4.0 - 4.1 changes though

File: ember-appkit-rails.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib,vendor}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency 'rails', '~> 4.0.0'
+  s.add_dependency 'rails', '~> 4'
```
Comment (bcardarella) [inline ember-appkit-rails.gemspec]: I would change this to be `'~> 4.0'` that way the versioning will stick to the minor

**Comment by abuiles** [comment]
@bcardarella @rjackson I didn't really check the internal changes in the generators between 4.0  and 4.1 but so far all is working fine. 

Working with minor makes sense, but then we would be restricted to that version of Rails :(, and 4.1 is almost out

**Comment by rwjblue** [comment]
@bcardarella - Our generators have decent test coverage. If you're good with it, I'll merge manually and add additional gem files to the Travis setup. That way we can run our suite on the versions we support.

Also, we should pin to `~>4.0` so that we do not update to 5.0 if/when it comes out.

**Comment by bcardarella** [comment]
@rjackson you don't mean Appraisal do you?

**Comment by rwjblue** [comment]
@bcardarella - No I meant with Travis' internal gemfiles option, but if you'd rather use Appraisal that would be fine with me too.

**Comment by abuiles** [comment]
@bcardarella @rjackson updated.

**Comment by bcardarella** [comment]
@rjackson ah, if there is a Travis way of doing it let's do that :+1:

**Pull request merged**